tasks.c

```diff
@@ -3854,7 +3856,7 @@ void vTaskSuspendAll( void )
         }
         else
         {
-            xReturn = xNextTaskUnblockTime - xTickCount;
+            xReturn = xConstNextTaskUnblockTime - xConstTickCount;
```

Comment:
Maybe xConstTickCount is enough to get ride of access multiple voilate variable in one statement. For example,
```c
xReturn = xNextTaskUnblockTime - xConstTickCount;
```

Reply:
Updated.